Barkane/.vsconfig

```diff
@@ -0,0 +1,6 @@
+﻿{
+  "version": "1.0",
+  "components": [
+    "Microsoft.VisualStudio.Workload.ManagedGame"
```

Comment:
Could you delete this file? I have it ignored on my end although not really sure if it has any side effects.

Reply:
Got it; it's just a Visual Studio config file. We should add it to gitignore if it's not necessary